pkg/event/target/scram_client.go

```diff
@@ -0,0 +1,36 @@
+package target
+
+import (
+	"crypto/sha256"
+	"crypto/sha512"
+	"hash"
+
+	"github.com/xdg/scram"
+)
+
+var SHA256 scram.HashGeneratorFcn = func() hash.Hash { return sha256.New() }
+var SHA512 scram.HashGeneratorFcn = func() hash.Hash { return sha512.New() }
+
+type XDGSCRAMClient struct {
+	*scram.Client
+	*scram.ClientConversation
+	scram.HashGeneratorFcn
+}
+
```

Comment:
some comments on what this file and why its needed? - also since this is a kafka specific file can this be renamed as kafka_scram_client.go?

Reply:
Updated